gettext-rs/src/getters.rs

```diff
@@ -91,6 +93,25 @@ pub fn domain_directory<T: Into<Vec<u8>>>(domainname: T) -> Result<PathBuf, io::
     }
 }
 
+/// Get currently set locale.
```

Comment:
Please document the returned value: why it can be `None` and how it's an opaque value rather than something immediately readable. You can just rework the sentence from `setlocale` I guess.

Reply:
Can it even be `None` at all? I read the man page three times and I'm still unsure, but it *should* only be possible when trying to *set* an invalid locale. Since we pass `null` in there, this should never happen and thus it is legal for us to `unwrap`?

Reply:
Apologies for the very delayed reply; I dropped the ball on this library a while back, but I'm getting back into it now.

I haven't looked at the C code yet, but in the "RETURN VALUE" section of the man page it's stated that "The return value is NULL if the request cannot be honored.". Our wrapper *must* to take it into account. The user might still decide to `unwrap()`, but it's their decision; we should only unwrap if we're 100% confident it's impossible to get an empty value.